vcxagency-artillery/easy-indysdk/README.md

```diff
@@ -0,0 +1,4 @@
+# Easy-Indysdk
```

Comment:
Could you instead reuse `easy-indysdk` in the root project to avoid duplication?

Reply:
I will remove vcxagency-artillery/easy-indysdk directory. 
It's not used but I did not remove it. My mistake.
Thanks.